kitsune/questions/jinja2/questions/includes/question_editing_frame.html

```diff
@@ -90,12 +90,20 @@ <h1 class="sumo-page-heading">{% block headline %}{% endblock %}</h1>
               {% endif %}
 
               <div class="info card shade-bg highlight mb">
+              {% if current_product.key == 'mozilla-account' %}
```

Comment:
shall we pass a flag from the view instead? The more we hardcode the value in multiple places, the harder will be to change it when needed. Ideally we should define that once in settings